### PR TITLE
feat(*): Revamp Handler

### PR DESCRIPTION
**Pull request by EvolutionX-10**

### CODE REVIEW

**Comment by EvolutionX-10** [comment]
@jacoobes you've started working on it again 👀 
just a suggestion, instead of `feat :` prefer doing `feat:`
a space can affect the way program reads commits

**Review by Allyedge — APPROVED** [review]
We already discussed this, you can merge the branch once the conflicts are solved.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "sern-handler",
-  "version": "0.1.0",
+  "name": "@sern/handler",
+  "version": "0.0.1",
   "description": "",
   "main": "dist/index.js",
```
Comment (Murtatrxx) [inline package.json:5]: Revert these changes, I did them in my branch

**Review by Murtatrxx — CHANGES_REQUESTED** [review]
Revert package.json & gitignore, changes, everything else is okay.

**Review by Murtatrxx — CHANGES_REQUESTED** [review]

**Comment by Murtatrxx** [comment]
Keep in mind, resolve conflicts should resolved after revamp-handler is successfully merged with revamp-docs (#47) and  #47 must merged before this.

**Comment by EvolutionX-10** [comment]
@jacoobes Closing the PR due to the discussion and solution mentioned in Discord

**Pull request closed without merging**